content/guides/tanzu-labs-engineer-faq/index.md

```diff
@@ -0,0 +1,326 @@
+
+Here's why: stories are planned, but bugs are an **unintended consequence of feature development** (and one that, as much as possible, we try to eliminate). A bug represents something that used to work, but now doesn't. You already got points for the work when it was a feature. You shouldn't get more points for breaking it and then fixing it again.
+
+Likewise, chores are a necessary aid to feature development, but typically aren't possible to plan up front—rather, they become apparent in reaction to the needs of the product as it evolves. If you start pointing bugs and chores, you'll artificially inflate your velocity.
+
+Imagine you've been pointing bugs and chores as they've cropped up and adding them into the backlog; also, imagine that process has led you to a current "velocity" of 20 points per week. Your backlog is now empty, and your product manager shows your team 10 new stories. You and the other engineers estimate those stories at 40 points. So you all say, "It will take us two weeks to complete these!" But you'll be wrong. Because you won't just work on those stories; as you develop those stories, you'll have to deal with bugs and chores that crop up. Since your velocity was artificially inflated, you weren't able to accurately predict how long the stories would truly take.
+
+Here's another way to think about it: bugs in tracker are reserved for something that used to work before, but now doesn't. Therefore, a bug is something the team already got points for when it was originally developed. It wouldn't be fair to point the bug and get points all over again—we'd be "rewarding" ourselves for breaking features that used to work! Instead, by not pointing bugs, we force our velocity to get dragged down by the bugs, since now we're spending time during the week fixing things we broke instead of delivering new value.
+
+Bugs and chores have to bring velocity down. That's why we don't point them. We want our velocity to tell us how long it will take to complete stories (i.e., user value). If you inflate it with bugs and chores, then you've lost your ability to predict how long it will actually take you to deliver new value.
+
+## Q: What's the value of CI/CD?
+
+There are two questions you have to ask when considering shipping software:
+* Can we ship?
+* Should we ship?
+
+"Should we ship?" is ultimately a business decision. Is it valuable to the business to put the latest features in the hands of the users right now? The product manager (PM) represents the business interests on the team and must own this decision.
+
+However, the question "Can we ship?" is fundamentally an engineering question. Is the software in a working state? Are we confident it won't fail in production? The goal of the XP engineers is to **always** have a "yes" answer to this question. A team that can't ship, can't learn. And the longer you're not learning, the greater the risk that you're wasting time and money building the wrong thing.
```

Comment:
Probably worth mentioning here, especially after all of the Log4J stuff recently, that a team that can't ship can't ship updated versions of libraries to fix vulnerabilities (or their own bugs!)

Reply:
Good suggestion -- will add.

Reply:
Updated.